api/users.js

```diff
@@ -33,6 +33,42 @@ module.exports = [
             return user;
         }
     },
+    {
+        method: 'GET',
+        path: '/users/{emailAddress}/emailExists',
+        config: {
+            auth: false,
+            validate: {
+                params: {
+                    emailAddress: Joi.string().email(),
+                }
+            }
+        },
+        handler: async (request, h) => {
+            const uow = await request.app.getNewUoW();
+            const logger = request.server.app.logger;
+
+            const emailAddress = request.params.emailAddress;
+
+            logger.debug(`Getting User by primary email address`);
+            logger.debug(`using email address: ${emailAddress}`);
+
+            let re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
We don't need to do the regex checking here. That's what Joi is doing above on line 43.